### PR TITLE
Fixed Administration redirect bug in UI /home

### PR DESCRIPTION
**Pull request by PetarNeshkov**
https://github.com/SoftUni-Internal/exam-systems-issues/issues/160

### CODE REVIEW

File: Servers/UI/OJS.Servers.Ui/OJS.Servers.Ui.csproj [hunk 1]
```diff
@@ -1,70 +1,78 @@
...
-            <ResolvedFileToPublish Include="@(DistFiles->'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
-                <RelativePath>%(DistFiles.Identity)</RelativePath>
-                <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
-                <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
-            </ResolvedFileToPublish>
-        </ItemGroup>
-    </Target>
-
-</Project>
+<Project Sdk="Microsoft.NET.Sdk.Web">
```
Comment (Minkov) [inline Servers/UI/OJS.Servers.Ui/OJS.Servers.Ui.csproj:1]: Changes in this file seem irrelevant.

**Review by Minkov — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/layout/content/PageContent.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ import SubmissionPage from '../../pages/submissions/SubmissionPage';
 import ContestPage from '../../pages/contest/ContestPage';
 import ContestResultsPage from '../../pages/contest-results/ContestResultsPage';
 import ContestsPage from '../../pages/contests/ContestsPage';
+import AdministrationPage from '../../pages/administration/AdministrationPage';
 import styles from './PageContent.module.scss';
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/layout/content/PageContent.tsx:13]: Can you please move this import on the bottom of all?

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/administration/AdministrationPage.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿import { useEffect } from 'react';
+import { useUrls } from '../../hooks/use-urls';
+
+
+const AdministrationPage = () => {
+    const { getAdministrationContestsGridUrl } = useUrls();
+
+    useEffect(() => {
+        window.location.replace(getAdministrationContestsGridUrl());
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/administration/AdministrationPage.tsx:9]: https://stackoverflow.com/a/7703736

**Review by shsimeonova — CHANGES_REQUESTED** [review]
